torch/_inductor/utils.py

```diff
@@ -912,7 +912,10 @@ def report_category(category, profile_events):
         for ev in profile_events:
             total_time += ev.self_cuda_time_ms
             percent = f"{ev.self_cuda_time_ms / wall_time_ms * 100:.2f}%"
-            rows.append([ev.key[:120], ev.self_cuda_time_ms, ev.count, percent])
+            key = ev.key
+            if key.startswith("triton_poi_fused_"):
```

Comment:
why this change?

Reply:
Some kernel name is too long to fit the screen. Remove the common/uninteresting prefix.